scripts/build/travis/configure_travis_trusty.sh

```diff
@@ -82,8 +82,8 @@ CXX_WARN_FLAGS="-Wall"
 #        C_CUSTOM_FLAGS="-fmessage-length=20"
 #        CXX_CUSTOM_FLAGS="-fmessage-length=20"
 # --------------------------------------------------------------------------------------------------------------
-C_CUSTOM_FLAGS=""
-CXX_CUSTOM_FLAGS=""
+C_CUSTOM_FLAGS="-flto"
```

Comment:
don't you want to use "thin-lto" instead? this takes an enormous amount of time and of memory